tests/unit/Parsers/ParserTestBase.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Tests\DataValues\Geo\Parsers;
+
+use DataValues\DataValue;
+use ValueParsers\ParseException;
+use ValueParsers\ValueParser;
+
+/**
+ * @deprecated
+ * TODO: remove
+ * This is a copy of ValueParserTestBase from DataValues Common,
+ * created so we can stop depending on that comment even though we
+ * did not refactor away the inheritance abuse here yet.
```

Comment:
I don't feel good merging a patch that contains a comment with such a negative wording. Labeling something as "abuse" like nobody could have an other opinion on that is just not nice.

Reply:
So me calling my own code bad is not nice? That is a somewhat surprising opinion to me, but to each his own.

What do you want to have done? Remove the comment, and all comments that say some code is problematic? Because someone can disagree about it being problematic, and problematic is a negative term. You can also change "abuse" into something else, but unless you remove the meaning from the todo, it will still say the code is "bad" and you can still disagree with it.

Reply:
I am specifically referring to the word "abuse": *"Treat with cruelty or violence, especially regularly or repeatedly"* ([source](https://en.oxforddictionaries.com/definition/abuse)).

Reply:
From the same source: Use (something) to bad effect or for a bad purpose; misuse.

Are you saying it is not clear what is the meaning meant in this context?

Reply:
I could say a lot here, but it essentially boils down to: please don't use bad words in code.

Reply:
Thiemo please answer my question. It is already clear you do not like the word used.

> What do you want to have done? Remove the comment, and all comments that say some code is problematic?

Reply:
I don't know what suggesting to "remove […] all comments that say some code is problematic" is meant to be, but it's not a question. Please approach me in person if you think there is something we can figure out here that wasn't already said.